mobile/src/libs/components/header/header.tsx

```diff
@@ -49,7 +51,9 @@ const Header: React.FC<Properties> = ({
       {isArrowVisible && <BackButton />}
 
       <View style={styles.titleBadgeContainer}>
-        <Text style={styles.title}>{title ?? name}</Text>
+        <Text style={isChat ? styles.chatTitle : styles.title}>
```

Comment:
```suggestion
        <Text style={[styles.title, isChat && styles.chatTitle]}>
```
Styles have similarities